### DISCUSSION TITLE
How do I change the background color of Visualizations? (Arch linux).

### DISCUSSION THREAD

**Discussion by Samueru-sama** [DeaDBeeF-Player/deadbeef#2898] [Q&A] [answered]
I can only change the base color for visualizations in the appearance tab in deadbeef, I don't have the background color option, searching about it, this was already asked (Issue #2779) and was apparently resolved, however I tried the lastest deadbeef and deadbeef-git from the Aur and no luck, I don't have the option.

Do I need to manually compile it to get it? Or is it a new bug?

**Comment by servimo** [comment, marked as answer]
https://github.com/DeaDBeeF-Player/deadbeef/issues/2779#issuecomment-1093115251

**Reply by Samueru-sama** [reply]
Ah, it seems I misread the conversation, so it is not there yet?

**Reply by Oleksiy-Yakovenko** [reply]
Yes, as stated in the issue (which is open) -- It has been implemented for Mac, but not for Linux version yet.

**Reply by Samueru-sama** [reply]
> Yes, as stated in the issue (which is open) -- It has been implemented for Mac, but not for Linux version yet.

Ok, sorry I thought that the small text saying that it was added to enhancement meant that it was done, I will have to wait then.

**Comment by Oleksiy-Yakovenko** [comment]
I have added global visualization background color configuration.
I'm also planning to add per-widget configurability, but this is not done yet.
